src/svg/qsvggenerator.cpp

```diff
@@ -72,13 +72,24 @@ static void translate_color(const QColor &color, QString *color_string,
     *opacity_string = QString::number(color.alphaF());
 }
 
-static void translate_dashPattern(const QVector<qreal> &pattern, qreal width, QString *pattern_string)
+static void translate_dashPattern(const QVector<qreal> &pattern, QString *pattern_string)
 {
     Q_ASSERT(pattern_string);
 
     // Note that SVG operates in absolute lengths, whereas Qt uses a length/width ratio.
     for (qreal entry : pattern)
-        *pattern_string += QString::fromLatin1("%1,").arg(entry * width);
+        /*
+        LF-37399
+        
+        Previously this function multiplied the entry position by the width, however
+        the line width was already being correctly set on the stroke-width SVG tag, 
+        multiplying these values increases the values inside the stroke-dasharray tag, 
+        which should not be happening because it increases the space seperation between 
+        the dashed lines, causing larger spaces when increasing the line width, distorting 
+        the image.
+        */
```

Comment:
I don't think the original behaviour described here is a bug. When you look at the onscreen rendering, increasing the line width does increase the dash spacing the match. That makes sense to me.